runner/parse.go

```diff
@@ -241,6 +241,10 @@ func newTaskFromSub(ctx Context, desc *SubTask, cfg *Config) (*Task, error) {
 		values[optName] = opt
 	}
 
+	if desc.Quiet {
+		subTask.Quiet = true
```

Comment:
What behavior would change if we removed this chunk of code?

My impression is that we would still be inheriting the sub-task setting because of the check for the `taskStack`, so I think there would be zero difference in behavior.

Reply:
Well, this specific mode wouldn't work:
```yaml
  quietSubtask:
    run:
      task:
        name: info
        quiet: true
  info:
    run: echo "Purely informative step ..."
```
You'd need to put a `quiet: true` in either the `info` task or the `quietSubtask`. But without the above chunk this `tusk.yml` will not be quiet. In other words, this is required for the taskStack check to work because otherwise we don't set this property on any task.

Or am I not understanding your question correctly?

Reply:
Anyway, I have removed this chunk aswell now. This leaves `quiet` properties on commands and tasks (with inheritance). I have adjusted the docs and tests accordingly.